Automattic/pocket-casts-ios
# Auto downloading episode: Global Settings UI

| 📘 Part of: [Autodownloading Project](https://github.com/orgs/Automattic/projects/1177)
|:---:|

Fixes #2296 

Adds the UI and settings to allow the selection the limit of downloads for the Auto download option

| 1 | 2 |
| - | - |
| ![Simulator Screenshot - iPhone 16 Pro - 2024-10-18 at 16 01 10](https://github.com/user-attachments/assets/f90a67f3-4cdf-46c0-bac6-2c5360064509) |  ![Simulator Screenshot - iPhone 16 Pro - 2024-10-18 at 16 01 13](https://github.com/user-attachments/assets/beb51cbe-5025-424c-9559-18de4c47b283) |

## To test

- Start the app
- Open the Profile tab
- Tap on the Cog / Settings
- Tap on Auto Download
- Enable the option New Episodes
- Check if the row with limits show
- Tap on the row
- Check if you see the options 1,2,3,5,10 episodes
- Select one
- Check if in the new screen the option is reflected
- Tap on the limits again
- Check if the current selection is show selected.

## Checklist
